prov/efa/src/rdm/efa_rdm_atomic.c

```diff
@@ -88,6 +88,58 @@ efa_rdm_atomic_alloc_txe(struct efa_rdm_ep *efa_rdm_ep,
 	return txe;
 }
 
+/**
+ * @brief Post atomic operations from give ep and tx entry
+ *
+ * @param efa_rdm_ep efa rdm ep
+ * @param txe tx entry
+ * @return ssize_t 0 on success, negative integer on failure
+ */
+ssize_t efa_rdm_atomic_post_atomic(struct efa_rdm_ep *efa_rdm_ep, struct efa_rdm_ope *txe)
+{
+	bool delivery_complete_requested;
+	int ret;
+	int req_pkt_type_list[] = {
```

Comment:
The original `static` qualifier kept the array in memory for the duration of the program (per process?); meaning it was only initialized once. It's difficult to know the actual impact of removing the `static` qualifier from a cursory glance; I just wanted to note that it was removed here.

Reply:
Added back